spring-integration-file/src/test/java/org/springframework/integration/file/config/FileOutboundChannelAdapterParserTests.java

```diff
@@ -175,6 +184,15 @@ public void adapterWithDirectoryExpression() {
 	}
 
 	@Test
+	public void adapterWithAppendNewLine() {
+		DirectFieldAccessor adapterAccessor = new DirectFieldAccessor(adapterWithAppendNewLine);
+			FileWritingMessageHandler handler = (FileWritingMessageHandler)
+				adapterAccessor.getPropertyValue("handler");
+		DirectFieldAccessor handlerAccessor = new DirectFieldAccessor(handler);
+		assertEquals(Boolean.TRUE, handlerAccessor.getPropertyValue("appendNewLine"));
```

Comment:
@artembilan This is great.  I've known about the Spring Integration test modules but never really gotten to explore much with them as-of-yet.  Thanks for the tip.